### ISSUE TITLE
[BUG]: Character set issues under Catalina 

### ISSUE THREAD

**Issue by ruudboon** [phalcon/cphalcon#14763]
Our Github runners run on the latest MacOS and show some issues with character conversion.
I was able to reproduce this on a local Mac as well. 

https://github.com/phalcon/cphalcon/runs/404587405#step:12:76

Expecting that we need to look at ext/kernel/filter.c - zephir_escape_multi()
Can you have a look at this @sergeyklay ?

Phalcon 4.0.2 - PHP 7.2 / PHP 7.3 / PHP 7.4

**Comment by ruudboon** [comment]
Steps to reproduce this. Run on MacOS Catalina with Phalcon 4.0.2
```
<?php

$source = 'émotion';
$expected = '\e9 motion';

$escaper = new \Phalcon\Escaper();
$result = $escaper->escapeCss($source);

echo "Source: " . $source . PHP_EOL;
echo "Expected: " . $expected . PHP_EOL;
echo "Result: " . $result . PHP_EOL;
if ($result != $expected) {
    echo "fail" . PHP_EOL;
} else {
    echo "ok" . PHP_EOL;
}
```
I think escapeCss is calling: https://github.com/phalcon/cphalcon/blob/master/ext/kernel/filter.c#L294

And then:
https://github.com/phalcon/cphalcon/blob/master/ext/kernel/filter.c#L149

**Comment by ruudboon** [comment]
This could maybe happening on windows as well. https://github.com/ruudboon/cphalcon/runs/409006716?check_suite_focus=true#step:28:380

**Comment by ruudboon** [comment]
@sergeyklay Let's leave this for now. I think other issues are currently more important.

**Comment by ruudboon** [comment]
This issue will be resolved with the next Zephir release. (0.12.17)

**Comment by sergeyklay** [comment]
Zephir 0.12.17 released

**Comment by ruudboon** [comment]
Fixed in 4.0.x
